src/app/views/homepage/homepage.component.html

```diff
@@ -49,14 +49,14 @@ <h1 class="card-header">
     <h2 class="text-center py-3">Quick Links</h2>
     <span class="col-12 col-md-4 offset-md-2 align-middle align-items-center ">
       <ul>
-        <li><a class="nav-link" routerLink="/events/monsoon-school">Monsoon Schools</a></li>
-        <li><a class="nav-link" routerLink="/events/workshop-and-meetings">Workshops</a></li>
-        <li><a class="nav-link" routerLink="/events/workshop-and-meetings">Seminars</a></li>
+        <li><a class="nav-link" routerLink="/events/monsoon-school" href="#" onclick="window.scrollTo(0, 0)">Monsoon Schools</a></li>
+        <li><a class="nav-link" routerLink="/events/workshop-and-meetings" href="#" onclick="window.scrollTo(0, 0)">Workshops</a></li>
+        <li><a class="nav-link" routerLink="/events/workshop-and-meetings" href="#" onclick="window.scrollTo(0, 0)">Seminars</a></li>
       </ul>
     </span>
     <div class="vr d-md-block d-none p-0"></div>
     <div class="col-3 d-md-block d-none offset-md-1 justify-content-center align-items-center" >
-      Opportunities don't happen, you create them.
+      "Opportunities are like sunrises. If you wait too long, you miss them."
```

Comment:
Did you check  the output on ui ?
Remove the quotes.
Did you check the new statement is lengthy hence it is looking bulging on smaller screens ?

Reply:
Yup I did the check, but that was asked so I did.

Reply:
Indeed. But it was suggested to put a lengthy one...so I did. but now reverted back.